modules/web/js/ballerina/model/tree-builder.js

```diff
@@ -58,6 +58,10 @@ class TreeBuilder {
         // 4. If there is a more specific node defined based on node kind we will use
         //    that object instead of generic object.
 
+        if (!treeNodes) {
+             treeNodes = requireAll(require.context('./tree/', true, /\.js$/));
+        }
+
```

Comment:
Had to take this in to the build function, because otherwise `tree-builder` can't be imported inside `compilation-unit-node`. The cyclic import results in `CompilationUnitNode` not being added to `treeNodes`